HashMap.java

```diff
@@ -0,0 +1,85 @@
+// Time Complexity : O(1)
```

Comment:
TC should be O(n) because in worst case you would be iterating through each node